src/MassTransit/Logging/Diagnostics/StartedActivity.cs

```diff
@@ -41,7 +41,7 @@ public void AddExceptionEvent(Exception exception, bool escaped = true)
                 { DiagnosticHeaders.Exceptions.Escaped, escaped },
                 { DiagnosticHeaders.Exceptions.Message, exceptionMessage },
                 { DiagnosticHeaders.Exceptions.Type, TypeCache.GetShortName(exception.GetType()) },
-                { DiagnosticHeaders.Exceptions.Stacktrace, ExceptionUtil.GetStackTrace(exception) }
+                { DiagnosticHeaders.Exceptions.Stacktrace, exception.ToString() }
```

Comment:
Why would this be changed?

Reply:
> Backends can use the language specified methodology for generating a stacktrace combined with platform information from the [telemetry sdk resource](https://opentelemetry.io/docs/specs/semconv/resource/#telemetry-sdk) in order to extract more fine grained information from a stacktrace, if necessary.

Reply:
I updated the value of `exception.stacktrace` to follow this [recommandation](https://opentelemetry.io/docs/specs/semconv/exceptions/exceptions-spans/#stacktrace-representation) and what is done in the [OTel SDK](https://github.com/open-telemetry/opentelemetry-dotnet/blob/dd877fe179f38c0d16a8e6793d7b8e036b3f19d3/src/OpenTelemetry.Api/Trace/ActivityExtensions.cs#L91%5D). As it is just a recommendation, I can revert this part if needed.

Reply:
The recommendation also states you can include more detail, which is what MassTransit does. So I'd like this part reverted to include the stack trace.

Reply:
done